.github/workflows/build.yml

```
@@ -281,6 +281,51 @@ jobs:
             ../scripts/xcodebuild.sh macos/TemplateExample.xcworkspace build
           fi
         working-directory: template-example
+  windows:
+    name: "Windows"
+    runs-on: windows-latest
+    strategy: 
+      matrix:
+        configuration: [Debug, Release]
+        platform: [ARM, x64, x86]
+    steps:
+      - name: Set up MSBuild
+        uses: microsoft/setup-msbuild@v1.0.1
+      - name: Setup VSTest.console.exe
+        uses: darenm/Setup-VSTest@v1
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install
+        run: |
+          yarn
+        working-directory: example
+      - name: Build bundle and create solution
+        run: |
+          yarn build:windows
+          node ../windows/test-app.js
```

We should use the installed script to ensure we didn't make any wrong assumptions:
```suggestion
          node node_modules/react-native-test-app/windows/test-app.js
```